config/kubernetes/sig-api-machinery/teams.yaml

```diff
@@ -16,6 +16,22 @@ teams:
     - lavalamp
     - liggitt
     privacy: closed
+  sig-api-machinery-members:
+    description: Members of sig-api-machinery
```

Comment:
an explanation of what this is used for may be helpful.

Reply:
It can be useful to have this group in the case SIG level voting needs to occur. @lavalamp is announcing today to update this file, so I figured I'd add this section so people can add themselves to it.